miscellaneous/build_templates/analysis.yml

```diff
@@ -102,7 +103,7 @@ jobs:
           DetectArguments: |
             --detect.project.name="product-readiness.$(bdProject)"
             --detect.project.version.name="$(Build.SourceBranchName)"
-            --detect.source.path=$(Build.SourcesDirectory)$(projPath)/
+            --detect.source.path="$(Build.SourcesDirectory)$(projPath)/"
```

Comment:
Would probably prefer that we rename the folder to `Data_Analysis` instead, the spaces are what I assume causes a problem.